Mathlib/Tactic/Relation/Symm.lean

```diff
@@ -8,42 +8,19 @@ import Std.Tactic.Relation.Symm
 import Lean.Elab.Tactic.Location
 
 /-!
-# `symm_saturate` tactic
+# `relSidesIfSymm?`
 
-For every hypothesis `h : a ~ b` where a `@[symm]` lemma is available,
-add a hypothesis `h_symm : b ~ a`.
+This may now be orphaned code which can be removed.
 -/
 
 set_option autoImplicit true
 
 open Lean Meta
 
-namespace Lean.MVarId
-
-/-- For every hypothesis `h : a ~ b` where a `@[symm]` lemma is available,
-add a hypothesis `h_symm : b ~ a`. -/
-def symmSaturate (g : MVarId) : MetaM MVarId := g.withContext do
-  let mut g' := g
-  let hyps ← getLocalHyps
-  let types ← hyps.mapM inferType
-  for h in hyps do try
-    let symm ← h.applySymm
-    let symmType ← inferType symm
-    if ¬ (← types.anyM (isDefEq symmType)) then
-      (_, g') ← g'.note ((← h.fvarId!.getUserName).appendAfter "_symm") symm
-  catch _ => g' ← pure g'
-  return g'
-
-end Lean.MVarId
-
 namespace Mathlib.Tactic
 
 open Lean.Elab.Tactic Std.Tactic
 
-/-- For every hypothesis `h : a ~ b` where a `@[symm]` lemma is available,
-add a hypothesis `h_symm : b ~ a`. -/
-elab "symm_saturate" : tactic => liftMetaTactic1 fun g => g.symmSaturate
-
```

Comment:
Has this been moved to Std, or just deleted? The Std commit log doesn't make any such move obvious.

Reply:
This is only used by `solve_by_elim`, and moved to Std as part of https://github.com/leanprover/std4/pull/447

Reply:
Ah sorry, I was foiled by "load diff" banners blocking by ability to Ctrl+F!